lang/en/block_opencast.php

```diff
@@ -263,17 +283,26 @@
 $string['hpublished'] = 'Published';
 $string['hvisibility'] = 'Visibility';
 $string['hprovide'] = 'Provide';
+$string['importvideos_settingmodeduplication'] = 'Duplicating Events';
+$string['importvideos_settingmodeacl'] = 'ACL Change';
 $string['importvideos_errornotenabledorworking'] = 'The import videos feature is not enabled or not working';
 $string['importvideos_importbuttontitle'] = 'Import videos';
 $string['importvideos_importheading'] = 'Import videos from another course';
 $string['importvideos_importjobcreated'] = 'The import of the selected videos into this course was scheduled. This process will happen in the background. You do not need to wait on this page. As soon as the process is started, the videos will appear in the <em>Videos available in this course</em> section';
+$string['importvideos_importjobaclchangedone'] = 'The import of the selected series with its videos into this course has been done. You can now access the videos from the list.';
+$string['importvideos_importjobaclchangeseriesfailed'] = 'Import failed, because the ACL change for seleceted series has failed.';
+$string['importvideos_importjobaclchangeseriesmappingfailed'] = 'Import failed, becasue mapping the series to the course has failed.';
+$string['importvideos_importjobaclchangealleventsfailed'] = 'Import has partially failed, because ACL change for all videos has failed.<br />However, the series is now mapped to this course and its ACL has been changed.';
+$string['importvideos_importjobaclchangeeventsfailed'] = 'Import has partially failed, because ACL change for some of the videos has failed.<br />However, the series is now mapped to this course and its ACL has been changed.<br />The ACL of following eventIDs could not the changed:';
 $string['importvideos_importjobcreationfailed'] = 'At least one of the videos could not be imported to this course. Please try again or contact your Moodle administrator.';
 $string['importvideos_importepisodecleanupfailed'] = 'At least one of the existing Opencast episode modules within this course could not be cleaned up. Please try again or contact your Moodle administrator.';
 $string['importvideos_importseriescleanupfailed'] = 'At least one of the existing Opencast series modules within this course could not be cleaned up. Please try again or contact your Moodle administrator.';
 $string['importvideos_processingexplanation'] = 'These video files will be duplicated in Opencast and then made available in this course.';
 $string['importvideos_sectionexplanation'] = 'In this section, you can import existing video files from other Moodle courses to this Moodle course.';
+$string['importvideos_aclprocessingexplanation'] = 'The seriesid of the selected course will be used in this course as secondary seriesid and the ACLs of its videos will be changed accordingly.';
+$string['importvideos_aclsectionexplanation'] = 'In this section, you can import existing video files from other Moodle courses to this Moodle course.';
```

Comment:
That's exactly the same string as 'importvideos_sectionexplanation'. Please delete it and use the existing one.